server/modules/scheduling/index.js

```diff
@@ -159,25 +195,27 @@ function getScheduleForInstance(instance) {
 
 function parseEnvironmentSchedule(environmentSchedule) {
   if (environmentSchedule.ManualScheduleUp === false && environmentSchedule.ScheduleAutomatically === false) {
```

Comment:
Yes I agree. I didn't touch that code here but I'll do it anyway. Worth making that a bit easier to read.